.github/workflows/MSBuild.yml

```diff
@@ -24,9 +25,19 @@ jobs:
     - name: Setup VSTest Path
       uses: darenm/Setup-VSTest@v1.2
 
+    - name: Load dotnet core 3.1.100 and 3.1.x
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: '3.1.x'
+      env:
+        DOTNET_MULTILEVEL_LOOKUP: false
+
```

Comment:
Not sure if we still need this old version. I think 6.0 is the oldest still supported version. We should probable enable 8.0 for the future.

Reply:
Updated to net6.0 for now